guides/anchor.md

```diff
@@ -354,7 +354,7 @@ You should get a response like:
 
 ## Compliance
 
-The final piece of the puzzle is handling regulatory compliance, like <abbr title="Know Your Customer">KYC</abbr>. The [Stellar compliance protocol](compliance-protocol.md) is a standard way to exchange compliance information and pre-approve a transaction with another financial institution.
+The final piece of the puzzle is handling regulatory compliance, like <abbr title="Anti-Money Laundering">AML</abbr>. The [Stellar compliance protocol](compliance-protocol.md) is a standard way to exchange compliance information and pre-approve a transaction with another financial institution.
```

Comment:
KYC is a more thorough process that a bank will go through when someone is creating a bank account with them. When they are sending a payment they just need enough info on the recipient to do the AML check to make sure the recipient isn't on some sanction list. Since people already have accounts in these scenarios, we are just dealing with AML.